x/superfluid/types/params.go

```diff
@@ -34,7 +33,7 @@ func DefaultParams() Params {
 	return Params{
 		RefreshEpochIdentifier: "day",
 		MinimumRiskFactor:      sdk.NewDecWithPrec(5, 2), // 5%
-		UnbondingDuration:      time.Hour * 24 * 21,      // 3w
+		UnbondingDuration:      Threeweeks,
```

Comment:
Lets switch to two weeks?

Reply:
yep

Reply:
I thought that it was three because of external unbondings or nah?

Reply:
Nah, it was an accident from it being written